input/examples/practitioner-darleen-mackay.xml

```diff
@@ -0,0 +1,30 @@
+<Practitioner xmlns="http://hl7.org/fhir">
+    <id value="darleen-mackay"/>
+    <meta>
+        <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
+    </meta>
+    <identifier>
```

Comment:
Local identifiers need to have assigner provided. And should say what kind of identifier they are. So could be EI.

Reply:
Not fixed